docs/ros/ros/api/sensors_api.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: ROS 2 Sensors API
+sidebar_label: Sensors API
+sidebar_position: 4
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+
```

Comment:
Should we add a brief info note somewhere at the top to indicate what the "#" means and how it is used in this context. Is the numbering convention consistent across reboots? Across multiple Huskys with the same config? Across different types of robots with the same sensors?

Reply:
Yes numbering will be consistent, other than the case of robots with built in sensors. This should be defined in the robot YAML section.

I will add a note on what "#" means.